### PR TITLE
[Crafting]Gold ring desynth

### PR DESCRIPTION
**Pull request by hooksta4**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?
Enables Gold Ring to be desynth'd.
https://www.bg-wiki.com/ffxi/Gold_Ring
https://ffxiclopedia.fandom.com/wiki/Gold_Ring?direction=next&oldid=740719

!setskill goldsmithing 60
!additem lightning_crystal
!additem gold_ring
Use lightning + ring to get NQ 1x gold ingot and HQ+ 2 Ingots.

## Steps to test these changes
image

### CODE REVIEW

File: sql/synth_recipes.sql [hunk 1]
```diff
@@ -4752,6 +4752,8 @@ INSERT INTO `synth_recipes` VALUES (75530,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,27
 INSERT INTO `synth_recipes` VALUES (75531,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,2773,2774,2774,2775,4161,0,0,2781,2781,2781,2781,1,1,1,1,'Enfeeb. Kit Sleep');
 INSERT INTO `synth_recipes` VALUES (75532,1,0,0,0,0,0,34,0,0,0,4100,4242,12994,0,0,0,0,0,0,0,818,818,848,848,2,3,1,2,'Shoes (desynth)');
 INSERT INTO `synth_recipes` VALUES (75533,1,0,0,0,0,25,0,0,0,0,4100,4242,12722,0,0,0,0,0,0,0,834,819,819,819,2,5,6,7,'Bracers (desynth)'); -- FFXIclopedia
+INSERT INTO `synth_recipes` VALUES (75534,1,0,0,0,60,0,0,0,0,0,4100,4242,13445,0,0,0,0,0,0,0,745,745,745,745,1,2,2,2,'Gold_Ring (desynth)'); -- https://www.bg-wiki.com/ffxi/Gold_Ring
```
Comment (Xaver-DaRed) [inline sql/synth_recipes.sql:4755]: 1,2,2,2 would imply:
- 1 ingot when NQ
- 2 ingots on any HQ.

That's not what BG wiki sais.

File: sql/synth_recipes.sql [hunk 2]
```diff
@@ -4752,6 +4752,8 @@ INSERT INTO `synth_recipes` VALUES (75530,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,27
...
+
```
Comment (Xaver-DaRed) [inline sql/synth_recipes.sql:4756]: This newline must be deleted

File: sql/synth_recipes.sql (hunk 1 above)
Comment (hooksta4) [reply sql/synth_recipes.sql:4755]: ah yes you are correct, hq1 is also 1 ingot..

**Comment by Xaver-DaRed** [comment]
https://github.com/LandSandBoat/server/pull/4446

**Pull request closed without merging**